argus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Automated Kubernetes monitoring
 name: argus
-version: 0.3.0
+version: 0.5.0
```

Comment:
Pls check if the version change is ok

Reply:
why are we skipping a minor version? Should be 0.4.0

Reply:
Actually, we had changed the minor version to 0.4.0 in https://github.com/logicmonitor/k8s-helm-charts/pull/28/files 
I do not know why the master branch still show the previous file